src/Sulu/Bundle/DocumentManagerBundle/Initializer/WorkspaceInitializer.php

```diff
@@ -35,7 +35,7 @@ public function __construct(ConnectionRegistry $registry)
      */
     public function initialize(OutputInterface $output)
     {
-        foreach ($this->registry->getConnections() as $connection) {
+        foreach ($this->registry->getConnections() as $name => $connection) {
             $workspace = $connection->getWorkspace();
```

Comment:
Seemed to work without using the AdminConnection in this Initializer, although I would not have expected it to.